lms/djangoapps/ccx/utils.py

```diff
@@ -128,6 +128,19 @@ def get_ccx_for_coach(course, coach):
     return None
 
 
+def get_ccx_for_coach_by_ccx_id(course, coach, ccx_id):
```

Comment:
the name can simply be `get_ccx_by_ccx_id`.

question: in this case the is `ccx_id` the `CustomCourseForEdX` primary key?